--- a/boot/dtbs/mt7622-bananapi-bpi-r64.dts	2021-09-15 09:27:26.875953005 +0200
+++ b/boot/dtbs/mt7622-bananapi-bpi-r64.dts	2021-09-15 09:23:25.597138058 +0200
@@ -1226,6 +1226,18 @@
      label = "lan3";
     };
 
+    port@5 {
+     reg = <5>;
+     label = "aux";
+     phy-mode = "rgmii";
+
+     fixed-link {
+      speed = <1000>;
+      full-duplex;
+      pause;
+     };
+    };
+
     port@6 {
      reg = <6>;
      label = "cpu";
