--- a/boot/dtbs/mt7622-bananapi-bpi-r64.dts	2021-09-15 09:27:26.875953005 +0200
+++ b/boot/dtbs/mt7622-bananapi-bpi-r64.dts	2021-09-15 09:23:25.597138058 +0200
@@ -1324,7 +1324,7 @@
  asm_sel {
   gpio-hog;
   gpios = <90 0>;
-  output-high;
+  output-low;
  };
 
 
@@ -1598,11 +1598,11 @@
 };
 
 &sata {
- status = "disable";
+ status = "enable";
 };
 
 &sata_phy {
- status = "disable";
+ status = "enable";
 };
 
 &spi0 {
